config/config.js

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+module.exports = {
+    development: {
+        username: process.env.DB_USER,
```

Comment:
change this configuration to use a database URL string rather provide a username, password, and database name. It too bulky to provide all this information via an environment variable.